### PR TITLE
Misc updates and cleanup

### PR DESCRIPTION
**Pull request by tas50**
Broken out with description of the changes in each commit

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+   To apply the Apache License to your work, attach the following
+   boilerplate notice, with the fields enclosed by brackets "[]"
+   replaced with your own identifying information. (Don't include
+   the brackets!)  The text should be enclosed in the appropriate
+   comment syntax for the file format. We also recommend that a
+   file or class name and description of purpose be included on the
+   same "printed page" as the copyright notice for easier
+   identification within third-party archives.
+
+Copyright 2013-2015, Tim Smith
```
Comment (jjasghar) [inline LICENSE]: *cough* This seems wrong ;)

Comment (tas50) [inline LICENSE]: copy paste fail

**Comment by guilhem** [comment]
Obviously :+1:  :)
@hufman ?

**Comment by guilhem** [comment]
I would like to see this PR merged before #210 v0.9.0 :)

**Pull request merged**